.changeset/pink-mayflies-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-newrelic-dashboard': minor
```

Comment:
Think this should be a `patch` change.  As there's no breaking change and the package is `<1.0.0`

Reply:
modified to patch